BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Public/Span.cs

```diff
@@ -254,16 +259,29 @@ internal void CalculateFrameRateMetrics(FrameMetricsSnapshot endFrameRateMetrics
                 span.SetAttributeInternal("bugsnag.span.category", "frozen_frame");
                 span.End(frameTimes.EndTime);
             }
+
+            var totalFrames = endMetrics.TotalFrames - beginningMetrics.TotalFrames;
```

Comment:
is it possible that `totalFrames` is `0`? we should probably guard against divide-by-zero